### ISSUE TITLE
TouchableOpacity causing Invalid props.style key `KhtmlUserSelect` supplied to `View`

### ISSUE THREAD

**Issue by respectTheCode** [necolas/react-native-web#97]
As soon as I add a TouchableOpacity component I start getting this warning every time it renders.

```
warning.js:45 Warning: Failed propType: Invalid props.style key `KhtmlUserSelect` supplied to `View`.
Bad object: {
  "KhtmlUserSelect": "none",
  "MozUserSelect": "none",
  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
  "WebkitTouchCallout": "none",
  "WebkitUserSelect": "none",
  "backgroundColor": "black",
  "cursor": "pointer",
  "msUserSelect": "none"
}
```

It looks like the invalid style keys are coming from `react-tappable`

**Comment by necolas** [comment]
Yep. I'm about to replace all the touchables to mirror RN, which will remove this warning.

**Comment by necolas** [comment]
Fixed by 3c4d7655db65c3ffaefda4b81a94dc5a7b33bfa6

**Comment by respectTheCode** [comment]
Thanks for the quick fix. That did it.
